build.gradle

```diff
@@ -154,6 +154,7 @@ shadowJar {
         from refMap
         manifest {
             attributes([
+                    "FMLAT"                      : accessTransformersFile.toString(),
```

Comment:
How shall we add it conditionally? Only when `accessTransformersFile.isEmpty() == false`?

Reply:
I am not sure how groovy arrays are handled, but you could probably define a variable like:
```
def manifestParams = [
"pram1WhichAlwaysPresent": "$param1Value",
"param2WhichAlwaysPresnet": "$param2Value",
...
]

and then just if and `+= ["...": "..."]` or `+= "...": "..."` or what ever syntax this has or migrate to kotlin and have clearly defined syntax of summing sets/maps ;p